packages/api/src/trpc-client.ts

```diff
@@ -3,36 +3,22 @@ import superjson from "superjson";
 import type { OpenApiMeta } from "trpc-openapi";
 import { ZodError } from "zod";
 
-import { logger } from "@blobscan/logger";
-
 import type { TRPCContext } from "./context";
-import { env } from "./env";
 
 export const t = initTRPC
   .context<TRPCContext>()
   .meta<OpenApiMeta>()
   .create({
     transformer: superjson,
     errorFormatter({ shape, error }) {
-      logger.info(error.cause?.message);
-
-      if (
-        error.code === "INTERNAL_SERVER_ERROR" &&
-        env.NODE_ENV === "production"
-      ) {
-        return { ...shape, message: "Internal server error" };
-      }
-
-      // TODO: Add sentry support
-      // if (process.env.NODE_ENV === "production") {
-      //   Sentry.captureException(error);
-      // }
```

Comment:
Don't we need sentry to capture the exception here?

Reply:
No, this for formatting the error only.  We notify sentry [here](https://github.com/Blobscan/blobscan/blob/bf9eb6ed72f8a88801d2d625e4b09f447299c4b8/apps/web/src/pages/api/trpc/%5B...trpc%5D.ts#L13) and [here](https://github.com/Blobscan/blobscan/blob/bf9eb6ed72f8a88801d2d625e4b09f447299c4b8/apps/rest-api-server/src/index.ts#L61)